lib/rules/no-send-action.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @fileOverview Disallow use of sendAction
+ * @author Edilberto Ruvalcaba
+ */
+'use strict';
+
+const { get } = require('../utils/get');
+
+function isSendActionFnCall(node) {
+  return get(node, 'callee.property.name') === 'sendAction';
+}
+
+const MESSAGE = 'Do not use send action. Consider using closure actions to work with JS functions instead of relying on the old action system. Please see following guide for more information: https://github.com/chadhietala/ember-best-practices/blob/master/guides/rules/no-send-action.md';
```

Comment:
This should probably reference `https://github.com/ember-best-practices/eslint-plugin-ember-best-practices/` instead of `https://github.com/chadhietala/ember-best-practices/`?